conan/cli/commands/graph.py

```diff
@@ -179,3 +197,75 @@ def graph_info(conan_api, parser, subparser, *args):
+                                "e.g: zlib/1.2.13:* means all binaries for zlib/1.2.13. "
+                                "If revision is not specified, it is assumed latest one.")
+
+    args = parser.parse_args(*args)
+    # parameter validation
+    validate_common_graph_args(args)
+
+    cwd = os.getcwd()
+    path = conan_api.local.get_conanfile_path(args.path, cwd, py=None) if args.path else None
+
+    # Basic collaborators, remotes, lockfile, profiles
+    remotes = conan_api.remotes.list(args.remote) if not args.no_remote else []
+    overrides = eval(args.lockfile_overrides) if args.lockfile_overrides else None
+    lockfile = conan_api.lockfile.get_lockfile(lockfile=args.lockfile,
+                                               conanfile_path=path,
+                                               cwd=cwd,
+                                               partial=args.lockfile_partial,
+                                               overrides=overrides)
+    profile_host, profile_build = conan_api.profiles.get_profiles_from_args(args)
+
+    if path:
+        deps_graph = conan_api.graph.load_graph_consumer(path, args.name, args.version,
+                                                         args.user, args.channel,
+                                                         profile_host, profile_build, lockfile,
+                                                         remotes, args.update,
+                                                         check_updates=args.check_updates,
+                                                         is_build_require=args.build_require)
+    else:
+        deps_graph = conan_api.graph.load_graph_requires(args.requires, args.tool_requires,
+                                                         profile_host, profile_build, lockfile,
+                                                         remotes, args.update,
+                                                         check_updates=args.check_updates)
+    print_graph_basic(deps_graph)
+    deps_graph.report_graph_error()
+    conan_api.graph.analyze_binaries(deps_graph, args.build, remotes=remotes, update=args.update,
+                                     lockfile=lockfile)
+    print_graph_packages(deps_graph)
+
+    ConanOutput().title("Retrieving and computing closest binaries")
+    # compute ref and conaninfo
+    missing = args.missing
+    for node in deps_graph.ordered_iterate():
+        if node.binary == BINARY_MISSING:
+            if not missing or ref_matches(node.ref, missing, is_consumer=None):
+                ref = node.ref
+                conaninfo = node.conanfile.info
+                break
+    else:
+        raise ConanException("There is no missing binary")
+
+    pkglist = conan_api.list.explain_missing_binaries(ref, conaninfo, remotes)
+
+    ConanOutput().title("Closest binaries")
+    return {
+        "Closest binaries": pkglist.serialize(),
```

Comment:
I think that this should be `closest_binaries` instead. It only represents the namespace for the closest binaries section, so I believe that it'd be better to use a variable following the Python syntax.